lib/modules/manager/bazel-module/bazelrc.ts

```diff
@@ -0,0 +1,140 @@
+          value = part.substring(equalSignIdx + 1);
+        } else {
+          name = part.substring(nameStartIdx);
+          const nextIdx = i + 1;
+          // Check for --option_name OR --option_name option_value
+          value =
+            nextIdx < parts.length && !parts[nextIdx].startsWith('--')
+              ? parts[nextIdx]
+              : undefined;
+        }
+        const option = new BazelOption(name, value);
+        options.push(option);
+      }
+    }
+    return options;
+  }
+}
+
+export class CommandEntry {
+  readonly entryType = 'command';
+  constructor(
+    readonly command: string,
+    readonly options: BazelOption[],
+    readonly config?: string
+  ) {}
+
+  getOption(name: string): BazelOption | undefined {
+    return this.options.find((bo) => bo.name === name);
+  }
+}
+
+type BazelrcEntries = ImportEntry | CommandEntry;
+
+function shouldProcessLine(line: string): boolean {
+  if (line.length === 0) {
+    return false;
+  }
+  if (line.startsWith('#')) {
+    return false;
+  }
+  return true;
+}
+
+function createEntry(line: string): BazelrcEntries | undefined {
+  const importResult = importRegex.exec(line);
+  if (importResult) {
+    const irGroups = importResult.groups!;
+    return new ImportEntry(irGroups.path, irGroups.type === 'try-import');
+  }
+  const optionResult = optionRegex.exec(line);
+  if (optionResult) {
+    const orGroups = optionResult.groups!;
+    return new CommandEntry(
+      orGroups.command,
+      BazelOption.parse(orGroups.options),
+      orGroups.config
+    );
+  }
+  return undefined;
+}
+
+export function parse(contents: string): BazelrcEntries[] {
+  return contents
+    .split('\n')
+    .map((l) => l.trim())
+    .filter(shouldProcessLine)
+    .map(createEntry)
+    .filter(isNotNullOrUndefined);
+}
+
+async function readFile(
+  file: string,
+  workspaceDir: string,
+  readFiles: Set<string>
+): Promise<BazelrcEntries[]> {
+  if (readFiles.has(file)) {
+    throw new Error(
+      `Attempted to read a bazelrc multiple times. file: ${file}`
+    );
+  }
+  readFiles.add(file);
+  const contents = await fs.readLocalFile(file, 'utf8');
+  if (!contents) {
+    return [];
+  }
+  const entries = parse(contents);
+  const results: BazelrcEntries[] = [];
+  for (const entry of entries) {
+    if (entry.entryType === 'import') {
+      const importFile = upath.normalize(
+        entry.path.replace('%workspace%', workspaceDir)
+      );
+      const importEntries = await readFile(importFile, workspaceDir, readFiles);
+      results.push(...importEntries);
+    } else {
+      results.push(entry);
+    }
+  }
+  return results;
+}
+
+export async function read(workspaceDir: string): Promise<CommandEntry[]> {
+  const bazelrcPath = upath.join(workspaceDir, '.bazelrc');
+  const readFiles = new Set<string>();
+  return (await readFile(bazelrcPath, workspaceDir, readFiles)).filter(
+    (entry): entry is CommandEntry => entry.entryType === 'command'
```

Comment:
Why add `command` and `import` to the result if you filter it here out again?    
I can not find a other usage of `readFile`.

Reply:
Good point. I updated `readFile()` to return `CommandEntry[]`.